### PR TITLE
Removed ring chart but added in local ranking

### PR DESCRIPTION
**Pull request by askiefer**
Addresses issue #78 and maybe #72? 

I took out the ring chart and added a local ranking text with a tooltip. The ring chart code is just commented out for now - @thfield I see that you added an issue to move it in to it's own file. 

Also changed the right hand text from Estar score text to local ranking text.

screen shot 2017-08-09 at 12 01 09 am

### CODE REVIEW

File: dashboard/estar.html [hunk 1]
```diff
@@ -148,12 +148,15 @@
 		          	<div class="row border-bottom">
 		          		<div class="col-xs-3">
 		          			<div class="row center">
-		          				<span class="circle-chart" id="energy-star-score-radial"></span>
+		          				<!-- <span class="circle-chart" id="energy-star-score-radial"></span> -->
+		          				<div class="local-ranking-container"><span class="local-ranking-lg">#<span class="building-ranking" class="local-ranking"></span></span><br>
+		          				out of 100<span class="center"><a data-toggle="tooltip" data-placement="right" title="The local ranking figure compares peer buildings by their ENERGY STAR Score. In cases where the rank is identical, a building's site EUI is also used. The higher a ranking, the better a building's performance.">  <i class="fa fa-info-circle" aria-hidden="true"></i></a></span></div>
```
Comment (thfield) [inline dashboard/estar.html]: The 100 in  "X out of 100" is variable now that this isn't an energy star score; but instead local ranking.  There's a change in #86 that takes care of this, but depending on which branch gets merged first, we should make sure the final product is something like  `out of <span class="local-ranking-length">100</span>`

File: dashboard/estar.html [hunk 2]
```diff
@@ -148,12 +148,15 @@
 		          	<div class="row border-bottom">
 		          		<div class="col-xs-3">
 		          			<div class="row center">
-		          				<span class="circle-chart" id="energy-star-score-radial"></span>
+		          				<!-- <span class="circle-chart" id="energy-star-score-radial"></span> -->
+		          				<div class="local-ranking-container"><span class="local-ranking-lg">#<span class="building-ranking" class="local-ranking"></span></span><br>
```
Comment (thfield) [inline dashboard/estar.html]: <span **class**="building-ranking" **class**="local-ranking">

File: dashboard/estar.html [hunk 3]
```diff
@@ -148,12 +148,15 @@
...
+		          				out of 100<span class="center"><a data-toggle="tooltip" data-placement="right" title="The local ranking figure compares peer buildings by their ENERGY STAR Score. In cases where the rank is identical, a building's site EUI is also used. The higher a ranking, the better a building's performance.">  <i class="fa fa-info-circle" aria-hidden="true"></i></a></span></div>
 		          			</div>
 		          		</div>
 			          	<div class="col-xs-7">
 			          		<div class="ranking-text">
-			          		In 2015, <span class="building-name">Hotel Zetta</span> earned an ENERGY STAR score of <span id="building-energy-star-score">43</span>. The national <span class="building-type-lower">hotel</span> median score is <span id="median-energy-star-score">50</span>.
+			          		In 2015, <span class="building-name">Hotel Zetta</span> had a local ranking of <span class="building-ranking">43</span> out of 100 among the <span id="total-building-type"></span> buildings from
```
Comment (thfield) [inline dashboard/estar.html]: same thing here

**Review by thfield — APPROVED** [review]
repeated `class` attributes that should be changed, but lgtm

**Comment by askiefer** [comment]
Awesome, thanks @thfield - I modified with your changes. I don't know whether we should keep the ring chart or not - on the one hang, I quite like it and it obviously adds a visual element otherwise missing but on the other hand idk if now it makes a whole lot of sense. What do you think? @eayoungs?

**Comment by thfield** [comment]
merge #97 before this to save separate ring chart code

**Comment by askiefer** [comment]
Fixed the merge conflicts on this one so it's ready (maybe be good to review it again @thfield ?) and then merge in #97 and then this.

**Comment by thfield** [comment]
I'll merge 97 and take another look at this

**Pull request merged**